CHANGES.md

```diff
@@ -13,6 +13,10 @@ Release Notes.
 * Support http headers collection for Gin
 * Support higher versions of grpc.
 
+### Bug Fixes
+* Fix panic error when root span finished.
+* Update the peer address is not required when creating exit span.
```

Comment:
If peer is absent, the error will occur at OAP end.

Reply:
I see the OAP side if the peer is absent(blank string), then I would have a strategy on this. It seems that sometimes the peer address cannot be obtained.

Reply:
If it can't be obtained, we should consider this is a plugin bug. 
OAP failove mechanism is just for avoiding errors, but logically it is still wrong.

This causes(for exit span)
1. No virtual service(DB, MQ, etcd.)
1. Trace broken at server side, as the trace context in sw8 header is wrong.
1. Topology broken in case <2>

In theory, when peer is absent, we should create an exit span.

Reply:
I have updated the error message when creating an exit span with an empty peer address. Then the exit span will not created.